### ISSUE TITLE
cart.availableShippingMethods when calling commerce/cart/get-cart no longer returns price for method

### ISSUE THREAD

**Issue by andrewhawkes** [craftcms/commerce#1312]
### Description
When you make an AJAX call to `commerce/cart/get-cart` the method objects in `cart.availableShippingMethods` no longer return the `price`

```
        "availableShippingMethods": {
            "nextDay": {
                "id": 2,
                "name": "APC Next Day - By 6 PM",
                "handle": "nextDay",
                "enabled": true,
                "isLite": false
            },
            "apcNextDayBefore9am": {
                "id": 4,
                "name": "APC Next Day - Before 9 AM",
                "handle": "apcNextDayBefore9am",
                "enabled": true,
                "isLite": false
            }
        },
```

This is after upgrading to the latest version of commerce from `2.2.17`

### Steps to reproduce

1. Call `commerce/cart/get-cart`
2. See response 

### Additional info

- Craft version: 3.4.9
- PHP version: 7.2.12
- Commerce: 3.0.11

**Comment by andrewhawkes** [comment]
As a bonus, I notice lots of properties are coming back with `priceAsCurrency`. That would be a nice bonus on the shipping methods.

**Comment by philipboomy** [comment]
I am curious about this too. I cannot find anything in the changelog (though that might just be me) but it would be nice to have the cost per method come back in the JSON response

**Comment by andrewhawkes** [comment]
@philipboomy it used to come back, my app was using it and now it's gone :(

**Comment by andrewhawkes** [comment]
Bump

**Comment by lukeholder** [comment]
Fixed in 4f139cd19c83eff78b36847016c4999672f98993 which will be out in Commerce 3.1 this week.

**Comment by andrewhawkes** [comment]
@lukeholder excellent, thanks!
